d2l-button-subtle.html

```diff
@@ -116,6 +116,7 @@
 			}
 		</style>
 		<button
+			aria-haspopup$=[[ariaHaspopup]]
```

Comment:
There is a requirement that screen readers read a slightly different label than the text. Could aria-label be included as well?

Reply:
I'm OK including `aria-label` and `aria-expanded`. We have to be careful though with things like `aria-labelledby` and `aria-describedby` since the ID references those attributes use won't work across shadow DOM boundaries.